app/services/zendesk_feedback.rb

```diff
@@ -0,0 +1,37 @@
+require 'zendesk_api'
+
+class ZendeskFeedback
+  @@zendesk_url = ENV['ZENDESK_URL']
+  @@zendesk_username = ENV['ZENDESK_USERNAME']
+  @@zendesk_token = ENV['ZENDESK_TOKEN']
+
+  def initialize
+    @client = ZendeskAPI::Client.new do |config|
+      config.url = @@zendesk_url
```

Comment:
Could we not pass the the environment variables here to avoid lines 4-6?

Reply:
We could indeed.